scripts/openshiftci-presubmit-e2etests.sh

```diff
@@ -12,5 +12,7 @@ make bin
 export PATH="$PATH:$(pwd)"
 export CUSTOM_HOMEDIR="/tmp/artifacts"
 
-make test-e2e-scenarios
+make test-e2e-beta
```

Comment:
wouldn't it be better to use `make test-e2e-all` to make sure that new e2e scenarios are executed once added?

Reply:
+1

Reply:
shouldn't this use `test-e2e-all`?